File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.cpp
class Solution {
public:
    int getMinDistance(vector<int>& nums, int target, int start) {
        int n=nums.size();
        int mindist=INT_MAX;
        for(int i = 0; i < n; i++)
        {
            if (nums[i] == target)
            {
                mindist=min(mindist,abs(i-start));
            }
        }
        return mindist;
    }
};

File: 1848-minimum-distance-to-the-target-element/1848-minimum-distance-to-the-target-element.cs
public class Solution {
    public int GetMinDistance(int[] nums, int target, int start) {
        var n=nums.Length;
        int mindist=int.MaxValue;
        for(int i = 0; i < n; i++)
        {
            if (nums[i] == target)
            {
                mindist=Math.Min(mindist,Math.Abs(i-start));
            }
        }
        return mindist;
    }
}